### PR TITLE
Timestamps refactoring, docs and tests.

### PR DESCRIPTION
**Pull request by AlexanderLitus**

### CODE REVIEW

File: client/src/main/java/org/spine3/protobuf/Timestamps.java [hunk 1]
```diff
@@ -38,53 +41,108 @@
...
-    public static boolean isAfter(Timestamp timestamp, Timestamp from) {
-        return (compare(from, timestamp) < 0);
+    /**
+     * Calculates if the {@code isAfter} timestamp is after the {@code point} timestamp.
+     *
+     * @param point the first point in time which is supposed to be before the {@code isAfter} timestamp
+     * @param isAfter the timestamp to check if it is after the {@code point} timestamp
+     * @return true if the {@code isAfter} timestamp is after the {@code point} timestamp, false otherwise
+     */
+    public static boolean isAfter(Timestamp point, Timestamp isAfter) {
```
Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/protobuf/Timestamps.java]: Did you check the usages of this method? From what I see the order of parameters changed.

Comment (alexander-yevsyukov) [inline client/src/main/java/org/spine3/protobuf/Timestamps.java]: Anyway, the previous naming and the order of parameters was consistent with the rest of methods in this class: the argument in question comes first. Please return the order back and change the naming accordingly.

**Comment by AlexanderLitus** [comment]
@alexander-yevsyukov, PTAL.

**Comment by AlexanderLitus** [comment]
OK, I've removed this method with default context. Anything else?

**Comment by alexander-yevsyukov** [comment]
Please change the min method name length to 2, and it'd be LGTM.

**Pull request merged**